Improve CLI Tables and IDs handling (#4241)
zenml-io/zenml: docs/book/reference/global-settings.md

┗━━━━━━━━┷━━━━━━━━━━━━┷━━━━━━━━┷━━━━━━━━━┷━━━━━━━━━━━━━━━━┷━━━━━━━━━━━━━━┛
```

{% hint style="info" %}

> The output format can be customized with `--output` (json, yaml, csv, tsv) and `--columns` flags. See [environment variables](environment-variables.md#cli-output-formatting) for details.

Reviewer: ```suggestion
The output can be customized with an `--output` (json, yaml, csv, tsv, table) option and a `--columns` selection. See [environment variables](environment-variables.md#cli-output-formatting) for more details.
```
